### PR TITLE
Fixes for IE/Edge

### PR DESCRIPTION
**Pull request by tsherif**
Big thing is fixing how accessors are handled without VAOs. Also fixed the examples for IE/Edge.

### CODE REVIEW

File: examples/getting-started/hello-instancing-low/app.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 /* global window, requestAnimationFrame, cancelAnimationFrame */
+import 'core-js';
```
Comment (Pessimistress) [inline examples/getting-started/hello-instancing-low/app.js]: I’d suggest importing this once in the website template.

File: examples/getting-started/external-context/app.js [hunk 2]
```diff
@@ -16,6 +16,11 @@ export default class AppAnimationLoop extends MiniAnimationLoop {
   start(props) {
     const canvas = this._getCanvas(props);
 
+    this.isDemoSupported = Boolean(canvas.getContext('webgl'));
```
Comment (Pessimistress) [inline examples/getting-started/external-context/app.js]: Under what circumstances would this be false?

Comment (tsherif) [reply examples/getting-started/external-context/app.js]: IE

File: examples/getting-started/hello-instancing-low/app.js (hunk 1 above)
Comment (tsherif) [reply examples/getting-started/hello-instancing-low/app.js]: I want the examples to run on their own in IE and Edge. Is it possible to do that via the HTML webpack plugin or something like that?

File: examples/getting-started/external-context/app.js (hunk 2 above)
Comment (Pessimistress) [reply examples/getting-started/external-context/app.js]: Like IE<11? I think it’s ok if we let the page crash.

File: examples/getting-started/hello-instancing-low/app.js (hunk 1 above)
Comment (Pessimistress) [reply examples/getting-started/hello-instancing-low/app.js]: I believe that’s what https://babeljs.io/docs/en/babel-preset-env#usebuiltins is for but we’ve had issues with it...

File: examples/getting-started/external-context/app.js (hunk 2 above)
Comment (tsherif) [reply examples/getting-started/external-context/app.js]: No IE 11 requires "experimental-webgl". I prefer having IE not work than complicating the example, but leaving the check out causes the whole site to crash.

File: examples/getting-started/transform-feedback/app.js [hunk 3]
```diff
@@ -50,13 +51,19 @@ void main() {
...
   }
 
   static getInfo() {
     return INFO_HTML;
   }
 
   onInitialize({gl}) {
+    this.isDemoSupported = isWebGL2(gl);
+    if (!this.isDemoSupported) {
+      return {};
```
Comment (Pessimistress) [inline examples/getting-started/transform-feedback/app.js]: Display a message?

File: examples/getting-started/external-context/app.js (hunk 2 above)
Comment (ibgreen) [reply examples/getting-started/external-context/app.js]: Just a thought - could these checks be moved into `AnimationLoop`? 

(Always looking for ways to minimize "clutter" in example code).

Comment (tsherif) [reply examples/getting-started/external-context/app.js]: I'm thinking the way examples are handled needs a bit of a refactor, especially now that there are examples that can't use `AnimationLoop`.

**Pull request merged**